### PR TITLE
fix: select root should not be focusable

### PR DESCRIPTION
**Pull request by scomea**
# Description
Select root had a tab index of -1 which meant it could be focused on as well as the trigger.  This caused the select to be focused twice and narrated twice in narrator scan mode.

## Motivation & context
Fixes narrator issues encountered by partners.

## Issue type checklist
- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/react/fast-components-react-base/src/select/select.tsx [hunk 1]
```diff
@@ -158,7 +158,7 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, Select
                 className={this.generateClassNames()}
                 onKeyDown={this.handleKeydown}
                 onClick={this.handleClick}
-                tabIndex={-1}
```
Comment (nicholasrice) [inline packages/react/fast-components-react-base/src/select/select.tsx:161]: won't this hide the element entirely from screen readers?

Comment (scomea) [reply packages/react/fast-components-react-base/src/select/select.tsx:161]: No, the actual focusable element is the trigger button contained within the component.

Comment (nicholasrice) [reply packages/react/fast-components-react-base/src/select/select.tsx:161]: I don't think this is the correct usage of aria-hidden, MDN explicitly recommends against doing exactly this (focusable element as child of `aria-hidden="true"`): https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute

It's also not clear to me why this helps address focusability  of the element - what problem is adding this attribute solving?

Comment (scomea) [reply packages/react/fast-components-react-base/src/select/select.tsx:161]: The point of the fix was to stop double narration but just removing the tab index seems to do it, that aria-hidden was redundant so removed it.

**Pull request merged**